client/a8c-for-agencies/sections/marketplace/checkout/style.scss

```diff
@@ -116,28 +114,23 @@
 	}
 
 	.checkout__summary-pricing-discounted {
-		font-size: 1.25rem;
-		font-weight: 600;
+		@include a4a-font-heading-lg;
 		color: var(--color-neutral-80);
-		line-height: 1;
 
 		@include break-xlarge {
-			font-size: 1.5rem;
-			font-weight: 700;
+			@include a4a-font-heading-xl;
```

Comment:
`font-weight` here is 700, and in the `a4a-font-heading-xl` it is 600. Is that ok?